examples/select-table-rows-by-content/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Select table rows</title>
+    <style>
+        .highlighted {
+            background-color: red;
+        }
+    </style>
+</head>
+<body>
+    <table>
+        <tr>
+            <th>Id</th>
+            <th>First Name</th>
+            <th>Last Name</th>
+        </tr>
+        <tr>
+            <td>1</td>
+            <td>John</td>
+            <td>Smith</td>
+        </tr>
+        <tr>
+            <td>2</td>
+            <td>Oliver</td>
+            <td>Black</td>
+        </tr>
+        <tr>
+            <td>3</td>
+            <td>Harry</td>
+            <td>Red</td>
+        </tr>
+    </table>
+
+    <script>
+        const rows = document.querySelectorAll('tr');
+
+        rows.forEach( row => { row.addEventListener('click', (e) => {
```

Comment:
IE11 doesn't have a `NodeList.forEach` method - https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach#Browser_Compatibility.

Also, IE11 doesn't support arrow functions (`() => {}`).
Since IE11 is supported browser do not use such functionality in examples.

Rewritten code:
```js
 const rows    = document.querySelectorAll('tr');
 const rowsArr = Array.prototype.slice.call(rows);

 rowsArr.forEach(function (row) { 
     row.addEventListener('click', function (e) {
         rowsArr.forEach(row => row.removeAttribute('class'));
         row.setAttribute('class', 'highlighted');
     })
 });
```

Reply:
edited